nbs/common.base_model.ipynb

```diff
@@ -33,14 +33,68 @@
     "import inspect\n",
```

Comment:
<p>Can you comment in the code why do we need this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/935/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Addressed in b5243ff